server/modules/analysis/repository/processingStepCalculationRepository.js

```diff
@@ -58,4 +66,20 @@ export const updateCalculationIndex = async (surveyId, processingStepUuid, index
   return queriesRes[queriesRes.length - 1]
 }
 
+export const updateCalculation = async (surveyId, calculation, client = db) =>
```

Comment:
i would use the same convention than insert. ```updateCalculationStep```

Reply:
or updateStepCalculation?

Reply:
ok